src/status_im/contexts/wallet/common/utils.cljs

```diff
@@ -87,6 +75,40 @@
        (map #(calculate-balance-for-token %))
        (reduce +)))
 
+(defn calculate-balance-from-tokens
```

Comment:
if possible that we cover these with tests would be great 👌

Reply:
Covered in the subs test 👍